src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/KafkaProducerFactory.cs

```diff
@@ -74,7 +74,7 @@ private string CreateKeyForConfig(ProducerConfig producerConfig)
         private IProducer<byte[], byte[]> CreateBaseProducer(ProducerConfig producerConfig)
         {
             var builder = new ProducerBuilder<byte[], byte[]>(producerConfig);
-            ILogger logger = this.loggerProvider.CreateLogger("Kafka");
+            ILogger logger = this.loggerFactory.CreateLogger(LogCategories.CreateTriggerCategory("Kafka"));
```

Comment:
Why are we changing from "Kafka" to LogCategories.CreateTriggerCategory("Kafka")?

Reply:
"Kafka" is the correct category name here. Have fixed this.